locales/en-US.json

```diff
@@ -296,33 +298,33 @@
   "username": "Username",
   "usernameRequired": "Username is required",
   "validation": {
+    "bothPasswordsSame": "Both passwords must be the save",
```

Comment:
There's a typo here which is fixed in master

Reply:
It's likely because this branch is based on an old master. I'll rebase it.